packages/medusa/src/api/routes/admin/gift-cards/index.ts

```diff
@@ -1,4 +1,6 @@
 import { Router } from "express"
+import "reflect-metadata"
```

Comment:
is this necessary?

Reply:
I am not sure, sometimes it seems that is nescessary and not having it will result in "Reflect.metadata is not a function". But tests are passing without it for gift cards, so I will remove it

Reply:
Tests failed during github actions without it so I am re-adding it.